cmd/apiserver/app/options/options.go

```diff
@@ -31,6 +32,7 @@ type ClusterPediaServerOptions struct {
 	Admission      *genericoptions.AdmissionOptions
 	//      Traces         *genericoptions.TracingOptions
 
+	Logs    *logs.Options
```

Comment:
`logs.Options`  looks better on top
```go
	SecureServing  *genericoptions.SecureServingOptionsWithLoopback
	Authentication *genericoptions.DelegatingAuthenticationOptions
	Authorization  *genericoptions.DelegatingAuthorizationOptions
	Audit          *genericoptions.AuditOptions
	Features       *genericoptions.FeatureOptions
	CoreAPI        *genericoptions.CoreAPIOptions
	FeatureGate    featuregate.FeatureGate
	Admission      *genericoptions.AdmissionOptions
	Logs               *logs.Options
```
 > It' s provided by kube

The usage position also needs to be changed.

Reply:
Thank you for your suggestion, I have made changes, please review